ogame/__init__.py

```diff
@@ -249,6 +274,18 @@ class resources:
 
         return resources
 
+    def build_time(self, id, resources):
```

Comment:
This is a function bedder suited on an different File named convenience or shortcut like in django.
All the convenience would be there that a bot dev would use all the time. __init__ is more for request

Reply:
price from constants would be moved there too